server/core/config.cc

```diff
@@ -593,6 +595,14 @@ static bool config_load_single_file(const char* file,
         }
     }
 
+    /* Check this after reading config is finished */
+    if ((gateway.writeq_high_water || gateway.writeq_low_water) &&
+        gateway.writeq_high_water <= gateway.writeq_low_water)
+    {
+        rval = -1;
+        MXS_ERROR("Invaild configuration writeq_high_water should greater than writeq_low_water");
```

Comment:
Some typos should be corrected.

Invaild -> Invalid
configuration -> configuration,
should greater -> should be greater